### DISCUSSION TITLE
Trims for Track also applied to Pre-Draw

### DISCUSSION THREAD

**Discussion by albansuser** [gpx-animator/gpx-animator#556] [General]
I frequently use the beginning and end trim function on Tracks. However these settings don't seem to apply to the "Pre-Draw" line. This, at least for my use, defeats the purpose of the trims and so I don't use Pre-Draw even though I'd like to.

Would it be possible to implement an option to have the trims also applied to Pre-Draws?

Thanks

PS I'm the user who requested implementation of title and end pages.

**Comment by McPringle** [comment]
Sure, this is a bug. I created an issue (#557) from this discussion and it will be fixed with v1.9. Thank you very much for reporting!
